### PR TITLE
Remove all unneeded set_include_path()

### PR DESCRIPTION
**Pull request by MorrisJobke**
- downstream of https://github.com/owncloud/core/pull/26468

I tested this and it still works - also enabling apps worked fine.

cc @LukasReschke @rullzer @nickvergessen @icewind1991

### CODE REVIEW

File: lib/base.php [hunk 1]
```diff
@@ -217,12 +217,7 @@ public static function initPaths() {
 
 		// set the right include path
 		set_include_path(
-			OC::$SERVERROOT . '/lib/private' . PATH_SEPARATOR .
-			self::$configDir . PATH_SEPARATOR .
-			OC::$SERVERROOT . '/3rdparty' . PATH_SEPARATOR .
-			implode(PATH_SEPARATOR, $paths) . PATH_SEPARATOR .
-			get_include_path() . PATH_SEPARATOR .
```
Comment (nickvergessen) [inline lib/base.php]: is this not required?

Comment (MorrisJobke) [inline lib/base.php]: We don't use any system wide installed classes - like via pear installed stuff

Comment (MorrisJobke) [inline lib/base.php]: We don't use any system wide installed classes - like via pear installed stuff

**Comment by LukasReschke** [comment]
LGTM

**Comment by MorrisJobke** [comment]
I tested it here again and it works 👍 -> merge

**Pull request merged**